lib/nexpose/scan_template.rb

```diff
@@ -577,5 +577,39 @@ def aces_level
     def aces_enabled?
       return 'full' == aces_level
     end
+
+    # Enable or disable the debug logging.
+    # @param [Boolean] enable Enable or disable the debug logging.
+    def enable_debug_logging=(enable)
+      return if enable.nil?
+      logging = REXML::XPath.first(@xml, 'ScanTemplate/Logging')
+      if logging.nil?
+        logging = REXML::Element.new('Logging')
+        @xml.add_element(logging)
+      end
+      debug_logging = REXML::XPath.first(logging, 'debugLogging')
+      if debug_logging.nil?
+        debug_logging = REXML::Element.new('debugLogging')
+        logging.add_element(debug_logging)
+      end
+      debug_logging.attributes['enabled'] = (enable ? 1 : 0)
+    end
+
+    # Enable or disable the enhanced logging.
+    # @param [Boolean] enable Enable or disable the enhanced logging.
+    def enable_enhanced_logging=(enable)
```

Comment:
I would expect this to be:

```
def enhanced_logging=(enable)
  debug_logging = enable
  aces_level = 'FULL'
end
```

Otherwise you have code duplication.